rasa/telemetry.py

```diff
@@ -631,7 +634,19 @@ def initialize_error_reporting() -> None:
         ],
         send_default_pii=False,  # activate PII filter
         server_name=telemetry_id or "UNKNOWN",
-        ignore_errors=[KeyboardInterrupt, RasaException, NotImplementedError],
+        ignore_errors=[
+            # std lib errors
+            KeyboardInterrupt,
+            MemoryError,
+            NotImplementedError,
+            asyncio.CancelledError,
+            # rasa errors
+            RasaException,
+            # 3rd party errors
```

Comment:
it might be better to define these closer to the classes that rely on pymongo / postgres / boto 🤔 that would make it easier to maintain (e.g. closer to channels, tracker store, etc...).

Reply:
Maybe there is a better approach: I am not sure raising these original exceptions directly is the best approach anyways. e.g. it might be better to define a `ConnectionException` error pointing to more information on where to fix an issue and use the original exception as a cause of this one (`raise ConnectionException(...) from NoCredentialsError`)